modules/showFun.js

```diff
@@ -0,0 +1,34 @@
+const booksUL = document.querySelector('.book-list');
+export default function show() {
```

Comment:
- [ ] Please, consider using arrow function syntax, as requested in the project 👍 
```
const show() => {
....
}
```